uspace/srv/hid/input/input.c

```diff
@@ -628,6 +628,9 @@ static void kbd_add_legacy_devs(void)
 #endif
 #if defined(UARCH_sparc64) && defined(PROCESSOR_sun4v)
 	kbd_add_dev(&chardev_port, &stty_ctl);
+#endif
+#if defined(UARCH_arm64) && defined(MACHINE_virt)
+	kbd_add_dev(&chardev_port, &stty_ctl);
```

Comment:
We should probably do this by passing the 'console' boot argument as we now do on other platforms. However, let's do this after the merge.

Reply:
I tried searching for the 'console' boot argument but with no luck. Could you please point me to the relevant code?

Reply:
For example, on sun4v, tools\ew.py adds this to the command line:

--prom-env boot-args="console=devices/\\hw\\pci0\\01:01.0\\com1\\a"

One can do [something similar](http://www.helenos.org/wiki/UsersGuide/UserSerialConsole#ia32andamd64) on x86.

On mips32/malta, the boot argument is hardcoded in the kernel as of now.

This boot argument is then copied to sysinfo by the kernel and userspace services such as the input and output services check for it using config_get_value("console").

Reply:
Thanks for the pointer, I will look into this.